### ISSUE TITLE
fix stringMoneyToCents()

### ISSUE THREAD

**Issue by cpjk** [BlueBadgerBois/badgerboi#14]
This function does not correctly return the amount of cents in a string like '123.33'. Derived from code @jboretsky wrote so I am assigning it to him.

https://github.com/BlueBadgerBois/badgerboi/blob/ea177d0a54405c53cb5903830cd274973270a214/server/handler.go#L137-L140

**Comment by cpjk** [comment]
Actually I think this works as long as there are two numbers after the decimal points always.

**Comment by jboretsky** [comment]
Yeah i was talking to Ed about it when I was writing that, It’s definitely not perfect, but we know all the inputs are gonna be formatted properly. How would you say we should convert something like “100.123” into cents? That 3 is a fraction of a cent, so we would just wanna chop it off probably?

**Comment by jboretsky** [comment]
Ah I just realized what you meant, yeah if we don't have 2 decimal pts behind a number then something like "$123" becomes 123 cents, so yeah we should probs make sure we have 2 decimal points after each number or something

**Comment by jboretsky** [comment]
Luckily for now it looks like the 1 user file is always formatted that way

**Comment by cpjk** [comment]
Yeah, not ideal, but it looks like we're fine for now. But for user inputs later when we need a UI, we might need to fix it to deal with poorly formatted input.
